src/widgets/attachments/attachments-overview.component.tsx

```diff
@@ -113,6 +162,7 @@ export default function AttachmentsOverview() {
             Delete
```

Comment:
I know this is existing code - sorry for being so picky (but I really want to have an eye on translations now); can you make it localizable, too?